### ISSUE TITLE
[A11y] Explore Collections - Grid - Page Title

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1896]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

Page title is "All Explore Collection".  This doesn't mean much to me.  I'm not sure if this is a failure, though - maybe it makes more sense to people who use the site more than I do.

### Suggested Fix

Maybe change page title, maybe not.  This one's definitely subjective.

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

Make sure page title makes sense

**Comment by jechols** [comment]
This needs to be examined, but may be one we close in order to figure out a better strategy to address page titles since most of them are a bit "off"
